Chaincase.SSB/DesktopTorManager.cs

```diff
@@ -29,5 +29,17 @@ public Task StartAsync(bool ensureRunning, string dataDir)
 			_torProcessManager.Start(ensureRunning, dataDir);
 			return Task.CompletedTask;
 		}
+
+		string ITorManager.CreateHiddenService()
+		{
+			// hidden service support for desktop may follow _torProcessManager
+			// update with @kiminuo's changes
+			throw new System.NotImplementedException();
```

Comment:
current _torProccessManager is in the Wasabi Subrepo. I'd rather not touch it with control methods just for SSB, especially since kiminuo such a great overhaul with a proper Socks TorControl that could be used.